src/fastpm.c

```diff
@@ -358,10 +358,10 @@ int run_fastpm(FastPMConfig * config, RunData * prr, MPI_Comm comm) {
         load_transfer_data(a_transfer, t_init);
     }
 
-    FastPMUSMesh * usmesh = NULL;
+    FastPMUSMesh *usmesh[FASTPM_SOLVER_NSPECIES] = {NULL};
```

Comment:
Is the usual particle array also an array of pointers (FastPMStore *), not an array of FastPMStore?